data-analytics/dataflow-python-examples/dataflow_python_examples/data_lake_to_mart.py

```diff
@@ -41,7 +41,7 @@ def __init__(self):
         self.schema_str = ''
         # This is the schema of the destination table in BigQuery.
         schema_file = os.path.join(dir_path, 'resources', 'orders_denormalized.json')
-        with open(schema_file) \
+        with open(schema_file)
```

Comment:
This should be a single line.

`with open(schema_file) as f:`

Reply:
Fixing to single line.